packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsList.tsx

```diff
@@ -40,20 +40,26 @@ const styles = (theme: ThemeType) => ({
   },
 });
 
-const shouldShow = (belowFold: boolean, curatedDate: Date, currentUser: UsersCurrent | null) => {
+const shouldShow = (atBottom: boolean, timeForCuration: boolean, currentUser: UsersCurrent | null, hasCurationDrafts: boolean) => {
   if (isEAForum) {
-    return !belowFold && currentUser?.isAdmin;
+    return !atBottom && currentUser?.isAdmin;
   } else {
-    const twoAndAHalfDaysAgo = new Date(new Date().getTime()-(2.5*24*60*60*1000));
-    return belowFold || (curatedDate <= twoAndAHalfDaysAgo);
+    return (atBottom === hasCurationDrafts) || timeForCuration;
   }
 }
 
-const SunshineCuratedSuggestionsList = ({ terms, belowFold, classes, setCurationPost }: {
+const hasCurationDrafts = (results: SunshineCurationPostsList[] | undefined): boolean => {
+  if (!results || results.length === 0) return false;
```

Comment:
I think you don't really need this line; `.some` evaluates to `false` if an array has no elements.

Reply:
Well, I at least need the `If (!results) ...` part

Reply:
Nah, you could just do `results?.some`, but not a big deal